custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -394,7 +394,7 @@ def validate(self, dataframe):
         passes = conditions_map[self.condition](n_columns, test_value)
         if not passes:
             return [
-                f"Num columns error, {n_columns} did not satisfy: {self.condition} {test_value}"
+                f"Num columns error, {n_columns} received but the schema says: {self.condition} {test_value}"
```

Comment:
```suggestion
                f"Incorrect column length. {n_columns} received. However, the schema dictates that columns should be {self.condition} {test_value}"
```

Reply:
How about 
f"Incorrect number of columns. {n_columns} received. However, the schema dictates that number of columns should be {self.condition} {test_value}"